dotnet/Capstone/Controllers/BirdController.cs

```diff
@@ -13,10 +14,12 @@ namespace Capstone.Controllers
     public class BirdController : ControllerBase
```

Comment:
Methods in this file are generally printing to the console rather than returning an error. I'm good to correct this in the database-rework branch, but I wanted to acknowledge that.

Reply:
I created an [issue](https://github.com/ahdisease/Bird-Finder/issues/8) for us to remember to correct this